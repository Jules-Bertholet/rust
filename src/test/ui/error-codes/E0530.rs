enum Enum {
    WithField(i32)
}

use Enum::*;


fn main() {
    match WithField(1) {
        WithField => {}
        //~^ ERROR E0530
    }
}
