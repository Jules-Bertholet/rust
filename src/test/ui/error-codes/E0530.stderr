error[E0530]: match bindings cannot shadow tuple variants
  --> $DIR/E0530.rs:10:9
   |
LL | use Enum::*;
   |     ------- the tuple variant `WithField` is imported here
...
LL |         WithField => {}
   |         ^^^^^^^^^
   |         |
   |         cannot be named the same as a tuple variant
   |         help: try specify the pattern arguments: `WithField(..)`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0530`.
